lib/datasource/remote/api/profile_repository.dart

```diff
@@ -37,8 +37,8 @@ class ProfileRepository extends HttpRepository with EosRepository {
 
   // TODO(Raul): Unify this code with _getAccountPermissions in guardians repo
   // Returns the first active key permission - String
-  Future<Result> getAccountPublicKey(String accountName) async {
-    print('[http] getAccountPublicKey');
```

Comment:
Not sure what this is for, or whether it would work - need to test or leave it as is.

Reply:
This is handling the case when an account has several keys. I put this in LWRE in Sept '22 when I ran across the issue and it has been working fine ever since. 
![image](https://github.com/JoinSEEDS/seeds_light_wallet/assets/2141014/0d3d805f-3167-4cb8-a7a4-eba550195af5)